### DISCUSSION TITLE
用量子電腦模擬蟲洞

### DISCUSSION THREAD

**Discussion by Aaron0810** [1101-GS4719-C/Python-Community-GS4719-C#1489] [科技新知]
https://www.epochtimes.com/b5/22/12/24/n13891309.htm

閒聊版的東西有點多來分享義典科技相關的事情

去年年底google團隊跟美國一些大學合作裡用量子電腦成功模擬出了以前人類無法完全解明的蟲洞，這算是一個重大突破，因為蟲洞的原理其實可以運用在通訊上，而他們是用量子糾纏實現的，當兩個量子彼特進入糾纏態後，只要不要進行任何改變單純觀測就好，兩個彼特之間就會互相影響，而不會受到距離限制，這可以運用在通訊方面非常方便，雖然現在都還在理論跟模擬階段，不過未來或許有機會使用量子電腦來傳訊息呢!

**Comment by youtinglin** [comment]
量子運算也是未來的一大科學發展趨勢，現階段的科學發展都須仰賴效能極高的運算系統，而量子電腦運算的技術，其功能強大，能耗也更低。是因為量子力學的特性，量子運算將減少運算時間，進而減少能量的消耗。以利於需要大量快速計算的模式模型的發展，甚至是近幾年氣候變遷對於人類的威脅，因此需更好的進行氣候預測的建模，利於調適和減災。

**Comment by ashleyy-huang** [comment]
還記得小時候Wi-Fi才剛起步，甚至還只是2/3G的年代，轉眼間已經是到了5G，各種通訊及網路設備都大大的更新，也改變了世界的運作。如果量子電腦真的誕生了那這個世界肯定又會是不一樣的面貌，人與人的溝通方式大大改變，希望可以在有生之年看到量子電腦的出現

**Comment by poweichen00** [comment]
量子糾纏的應用可能在未來帶來非常多有趣的可能性，可以讓全球通訊更快速、更安全，希望之後可以看到。
